README.md

```diff
@@ -112,6 +118,22 @@ As a bonus, you can specify a **specific version**:
 $ bpkg install jwerle/suggest.sh@0.0.1 -g
 ```
 
+Install from github directly
+```sh
+bpkg install "https://github.com/{user}/{project}"
```

Comment:
Why include the host here?

Reply:
When the host is not included, the line below will work
```sh
BPKG_REMOTES[0]=${BPKG_REMOTE-https://raw.githubusercontent.com}
```

It means the default behavior is still supported. And you can install from github as
```sh
bpkg install jwerle/suggest.sh@0.0.1 -g
```

For this line
```sh
bpkg install "https://github.com/{user}/{project}"
```

It will not work until I refactored it as the parsed repo and url is incorrect

Reply:
OK. No problem.